framework/test/unit/modules/interoperability/mainchain/endpoint.spec.ts

```diff
@@ -90,6 +95,20 @@ describe('MainchainInteroperabilityEndpoint', () => {
 			);
 		});
 
+		it('should throw error if name has 0 length', async () => {
```

Comment:
This should be a test that exceeds limit right?
Also `'dummy'.repeat(MAX_CHAIN_NAME_LENGTH + 1)` would have a length of `5*(MAX_CHAIN_NAME_LENGTH + 1)`. Should simply change to `"a".repeat(MAX_CHAIN_NAME_LENGTH + 1)`